contents/product-engineers/behavioral-analytics.mdx

```diff
@@ -0,0 +1,117 @@
+}  
+---
+
+import { ProductScreenshot } from 'components/ProductScreenshot'
+import ReplayLight from '../images/product-engineers/behavioral-analytics/replay.png'
+import ReplayDark from '../images/product-engineers/behavioral-analytics/replay-dark.png'
+
+The measurement and analysis of user behavior, AKA behavioral analytics, is critical to building  successful products. This has long been a responsibility of product managers, marketers, and analysts, but that is changing. Many engineers are [taking on product responsibilities](/blog/what-is-a-product-engineer), and with that, are developing a need for behavioral analytics.
+
+We know because we build behavioral analytics tools, like [product analytics](/product-analytics) and [session replay](/session-replay), for these engineers. We've seen a growing trend of engineers using analytics to track user experience, measure the impact of their work, and discover areas needing improvement.
+
+We also know that engineers are busy and behaivoral analytics is, at most, a small part of their job. To help you get the most value out of behavioral analytics as fast as possible, this guide covers the most practical and relevant parts of behavioral analytics for engineers.
+
+## 1. Start small
+
+Most guides recommend doing a lot of planning before implementing behavioral analytics. They encourage you to "map your customer journey" and develop a foolproof "tracking plan". We disagree. Strongly, in fact.
+
+Why?
+
+Because striving for a "perfect" plan is just a recipe for frustration. There's so much you _can_ track and often the most the most valuable things are the least predictable. We're in the "[jump off the cliff and build an airplane on the way down](https://twitter.com/reidhoffman/status/1481071139221049345?lang=en)" camp. You'll just learn faster this way.
+
+![Real behavior](../images/product-engineers/behavioral-analytics/real_behavior.png)
+
+Start by tracking one specific area and expanding from there — e.g. track the usage of a specific feature. If you're using PostHog you can either:
+
+1. Rely on PostHog's [autocapture](/docs/product-analytics/autocapture) and create an [action](/docs/data/actions) representing usage.
+
+2. Capture a [custom event](/docs/getting-started/send-events#2-capture-custom-events). Customer events we track include `insight analyzed`, `dashboard created`, and `survey launched`.
+
+3. A bit of both — we recommend this.  
+
+Starting small enables you to get the basics implemented and begin to develop a sense of the value of data. You can create [trends](/docs/product-analytics/trends), breakdowns, [retention charts](/docs/product-analytics/retention), and more with just a single data point. Once you grasp this, it reveals other data points you'd like to include and you can build on what you've done.
```

Comment:
Should we add something about how you can be useful to create a tracking plan once you've got a feel for the basics? Totally optional.

Reply:
I think this is a whole can of worms that I don't want to open in this piece, I'm not fully sure how to create a great tracking plan and don't think it's necessary (for now) so just leaving it as is.